src/lib/theme/_theme.scss

```diff
@@ -4,28 +4,30 @@
 
 // @import 'ridley-grotesk'; // Uncomment when embedded font file is present
 
+// TODO: Reviseef
 // Teradata Font
 $td-font-default: 'Arial, Helvetica, sans-serif';
-$td-font-regular: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
-$td-font-bold: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
-$td-font-semibold: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
-$td-font-medium: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
+$td-font-regular: '"RidleyGrotesk", "RidleyGrotesk-Regular", Arial, Helvetica, sans-serif';
+$td-font-medium: '"RidleyGrotesk", "RidleyGrotesk-Medium", Arial, Helvetica, sans-serif';
+$td-font-semibold: '"RidleyGrotesk","RidleyGrotesk-Semibold", Arial, Helvetica, sans-serif';
+$td-font-bold: '"RidleyGrotesk","RidleyGrotesk-Bold", Arial, Helvetica, sans-serif';
```

Comment:
Should we name these $td-font-display-[weight] so it's clear? the difference between regular and default isn't super obvious.

Reply:
done